<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlgClass</class>
 <widget class="QDialog" name="LoginDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>431</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>431</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>南通大学医学院实验室物品管理系统</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>0</y>
     <width>441</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Labms_v1.qrc">:/images/Resources/南通医学院.bmp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>连接数据库(Connect to MySQL)</string>
   </property>
   <widget class="QLineEdit" name="m_name">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>47</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="m_pwdLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>73</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_nameLabel">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>47</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserName:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_password">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>73</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Host:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_host">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>21</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>23</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_port">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>19</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_CancelBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>取消登录并关闭窗口</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>确定登录</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Labms_v1.qrc"/>
 </resources>
 <connections/>
</ui>
